packages/v4/src/content/developer-resources/accessibility-testing.md

```diff
@@ -0,0 +1,106 @@
+
+- **Dynamic content** - One of the biggest accessibility concerns with dynamic content is the need to notify users that the content has changed. Sighted users benefit from highlighting or drawing visual attention to the changes. Non-sighted users need to be notified some other way, such as by loading a new page, sending the focus to the new content, or using ARIA live announcements.
+
+- **Unrelated announcements** from the OS outside of the web app - Other notifications can interrupt a user while interacting with a page. For example, "you received a new chat msg from xyz". This possibility should be considered when designing and developing for screen readers as it will impact their experience. This is why it's good to keep things like validation messages and alerts terse, as it makes it less likely that an announcement will be interrupted. 
+
+
+<br />
+
+### Checklist for testing
+These are some of the main areas we check for on the PatternFly team. We recommend that you check for these same areas in your applications, too. 
+
+**General questions**
+
+<input type="checkbox" unchecked /> VO Rotor Navigation can discover all information <br/>
+<input type="checkbox" unchecked /> VO Shortcut Navigation can discover all information <br/>
+<input type="checkbox" unchecked /> VO Cursor Navigation can discover all information. (dependent on component and use case- easy way to navigate to component) <br/>
+<input type="checkbox" unchecked /> Tabbed Keyboard Navigation can discover all information
+- However, if content should be hidden from a screen reader, for instance, if it’s offscreen or just presentational, make sure that content is set to aria-hidden=”true”.  <br/>
+
+<input type="checkbox" unchecked /> Is it understandable and usable? <br/>
+  - <input type="checkbox" unchecked /> When you navigate to an item by keyboard or by screen reader, you can easily understand and use the item <br/>
+  - <input type="checkbox" unchecked /> Flow of information makes sense when navigating - screen readers navigate the page in DOM order, if you’ve used CSS to visually reposition elements, they may be announced in a nonsensical sequence. If you need something to appear earlier in the page, try to physically move it earlier in the DOM.
+
+**Accessibility points** <br />
+<input type="checkbox" unchecked /> **Structure:** Need to map the visual information architecture to the various rotor menus that exist by default. <br />
+- <input type="checkbox" unchecked /> **Check rotor:** are there good headings, landmarks, links, form controls, etc? Do they make sense, are descriptive but not overloading with info? <br />
```

Comment:
```suggestion
- <input type="checkbox" unchecked /> **Check rotor:** Are there good headings, landmarks, links, form controls, and other elements? Do they make sense, and are they descriptive without overloading information? <br />
``` 
I've been replacing or removing "etc" and "ie" because those abbreviations aren't easily understood by global audiences, especially for folks whose native language isn't rooted in Latin. If my replacement for "etc" doesn't make sense here, definitely feel free to replace it with a different word!